From 51d2a3e79a1770b6d4f67c1e36e1adddf1fc9228 Mon Sep 17 00:00:00 2001
From: Dennis Bonke <admin@dennisbonke.com>
Date: Thu, 23 Sep 2021 23:56:13 +0200
Subject: [PATCH] Don't build the documentation

xmlto is required to build the documentation, and that adds almost 800 MB of disk space for my host alone, likely more so for the rootfs. This is unacceptable, so we remove it.

Signed-off-by: Dennis Bonke <admin@dennisbonke.com>
---
 Makefile | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 87966a9..f5e6578 100644
--- a/Makefile
+++ b/Makefile
@@ -90,7 +90,6 @@ endif
 all: $(LIBGIFSO) libgif.a $(LIBUTILSO) libutil.a $(UTILS)
 ifeq ($(UNAME), Darwin)
 else
-	$(MAKE) -C doc
 endif
 
 $(UTILS):: libgif.a libutil.a
@@ -132,7 +131,7 @@ reflow:
 ifeq ($(UNAME), Darwin)
 install: all install-bin install-include install-lib
 else
-install: all install-bin install-include install-lib install-man
+install: all install-bin install-include install-lib
 endif
 
 install-bin: $(INSTALLABLE)
-- 
2.45.0

